apps/ais/src/app/app.module.ts

```diff
@@ -40,16 +40,16 @@ export class MyHammerConfig extends HammerGestureConfig {
   };
 }
 
-const regexp = /^([^(]*)(.*)/i;
-function profileTitleFormatter(title: string): string {
-  const match = title.match(regexp) || [];
-  if (match.length !== 3) {
-    throw new Error(
-      `Failed to transform the profile title "${title}" with the supplied profile formatter`
-    );
-  }
-  return `<i>${match[1]}</i>${match[2]}`;
-}
+// const regexp = /^([^(]*)(.*)/i;
```

Comment:
needless to say: vor dem Merge noch die Kommentare entfernen.

Reply:
Ja natürlich. Ich lasse da um sicherzustelln, das stimmt wohl und du bestätigst das bevor ich es komplett lösche und dann merge.